### ISSUE TITLE
Drivers and Programs

### ISSUE THREAD

**Issue by LucianoJulio** [MindShow/USBDisplay#85]
The system is very slow for the version of Windows 10, version 22H2, and the USB driver that appears for me is MS USB2.0 Display. I need the USB3.0 driver version.
The version of the program installed is 2.0.1.7.3, which is the only version that works precariously in Windows with great slowness and without the mouse cursor appearing.
![Geren](https://github.com/MindShow/USBDisplay/assets/43302965/4ed98a64-edc4-4fa1-9b96-4664149158fa)
![Gen2](https://github.com/MindShow/USBDisplay/assets/43302965/d57904dc-4dc2-42a5-b801-d787e0341cc1)

**Comment by DSergeev17606** [comment]
How it works with 3.0.1.40?
http://www.ultrasemi.com/usbdisplay/

**Comment by LucianoJulio** [comment]
For this version of the driver, installation was normal. The USB driver was installed with the same version 2.0. But the system (program) was not started by Windows, even after two reboots.
The program that displays this screen was not started.
![Gen2](https://github.com/MindShow/USBDisplay/assets/43302965/bdd04a3f-1c73-4d0e-8073-8c9a60d4b675)
As a result, the monitor did not display the images. The device was detected in the Windows screen manager, but it still didn't work because the program wasn't loaded during startup.

**Comment by DSergeev17606** [comment]
Are you sure it's the USB 3.0 version?
I bought it with 3.0, but it turned out to be 2.0. (MS912x chip)

**Comment by LucianoJulio** [comment]
In fact, I also bought it with 3.0, but it looks like 2.0. I couldn't identify the chip. Is there a driver or program that actually makes the equipment work? Thanks for your help.

**Comment by xbotuk** [comment]
use USB3.2 port

**Comment by s-kimball** [comment]
Is there any chance of getting the source code? I don't trust the driver and it does show a couple malicious (AI/ML derived threats) possibly false positives, but hard to tell. I did a quick test on an offline machine for 30 min and the most recent driver on ultrasemi's site works with a usb 3.0 port (800x600 for flight sim MFD). If you're getting a bunch of lag, it's probably a bandwidth limitation. don't expect these things to do 4k, and even 1080p/60 might be hit or miss. 

For reference the ones I got have the MS9122 chip in them. I really would like to see an open driver or a signed driver from another legit company using the same chip.

**Comment by LucianoJulio** [comment]
Thanks for the USB 3.2 recommendation, but my notebook only has 2.0 and 3.0 ports. The option now is to wait for the manufacturer to provide a new driver and functional program for Windows 10.

**Comment by DSergeev17606** [comment]
Yes. I've given up on these devices. The implementation is too "crooked". In favor of Displaylink. They work much better and there are no major problems.
